R/summary.pop_data.R

```diff
@@ -12,21 +13,40 @@
 #' library(dplyr)
 #'
 #' xs_data <- load_pop_data("https://osf.io/download//n6cp3/")
+#' summary(xs_data, strata = "Country")
 #'
-#' summary(xs_data)
-#'
-summary.pop_data <- function(object, ...) {
+summary.pop_data <- function(object, strata = "Country", ...) {
```

Comment:
I'm not sure we should default to stratifying by "Country"; I'd think the default should be `NULL`? @kaiemjoy  please advise

Reply:
I have set strata as `NULL` and have the old summary when null and provide stratification by the provided `strata`